app/views/layouts/govuk_admin_template.html.erb

```diff
@@ -108,6 +108,10 @@
           ga('set', 'anonymizeIp', true);
           ga('send', 'pageview');
         </script>
+      <% else %>
+        <script>
+          window.ga = function() {console.log.apply(console, arguments);}
```

Comment:
`console.log` isn't present in all browsers – eg IE7, this could break some JS when testing in those browsers.
http://caniuse.com/#feat=console-basic. We could check for the console functionality beforehand.